riotbuild/Dockerfile

```
@@ -309,6 +312,12 @@ RUN \
         true; \
     done"
 
+RUN \
+    echo 'Installing C2Rust' >&2 && \
+    CARGO_HOME=/opt/rustup/.cargo cargo install --no-track --locked c2rust --git https://github.com/immunant/c2rust --rev ed56c794230bbf5a0e417941e4778316d31ab114 && \
+    echo 'Cleaning up root-owned crates.io cache' >&2 && \
```

Maybe set the rustup version in an ARG ?

```suggestion
ARG RUSTUP_VERSION=ed56c794230bbf5a0e417941e4778316d31ab114
RUN \
    echo 'Installing C2Rust' >&2 && \
    CARGO_HOME=/opt/rustup/.cargo cargo install --no-track --locked c2rust --git https://github.com/immunant/c2rust --rev $RUSTUP_VERSION && \
    echo 'Cleaning up root-owned crates.io cache' >&2 && \
```

---

I expect the next change on this to be switching from a commit version to a cargo released version, which would then be a different line (no --git, no --rev) again. Then it'd make sense, now I see little value in it.

---

Ok I see, let's leave it like this then!